vllm/outputs.py

```diff
@@ -84,7 +84,7 @@ def __init__(
     ) -> None:
         self.request_id = request_id
         self.prompt = prompt
-        self.prompt_token_ids = tuple(prompt_token_ids)
+        self.prompt_token_ids = list(prompt_token_ids)
```

Comment:
do we need to do this?

Reply:
The test case corresponding to this change is:
```python
        for req_output in req_outputs:
            ...
            prompt_ids = req_output.prompt_token_ids
            ...
            for sample in req_output.outputs:
                ...
                req_sample_output_ids.append(prompt_ids + output_ids)
                ...
            ...

```
And the code `prompt_ids + output_ids` requires `prompt_ids` to be list.

I guess processing vllm's output using code like `prompt_ids + output_ids` may be common in current vllm-based apps. So maybe keeping the `prompt_token_ids` attribute in `RequestOutputs` as list would be better for compatibility's sake.

Reply:
Hmm I think it would be fine to keep it as tuple and cast it to list in the test code tbh